.github/labeler.yml

```diff
@@ -0,0 +1,43 @@
+
+c/bindings:
+  - any: ['*Bindings*/*', '!*Bindings*Tests/*']
+
+c/common:
+  - any: ['*Common*/*', '!*Common*Tests/*']
+
+c/connectivity:
+  - any: ['*Connectivity*/*', '!*Common*Tests/*']
+
+c/permissions:
+  - any: ['*Permissions*/*', '!*Permissions*Tests/*']
+
+c/push-notifications:
+  - any: ['*Permissions*/*', '!*Permissions*Tests/*']
```

Comment:
Looks like incorrect projects are mentioned here - should be PushNotifications

Reply:
Awesome!